sdk/communication/Azure.Communication.Sms/CHANGELOG.md

```diff
@@ -5,7 +5,13 @@
 ### Added
 - Added support to create SmsClient with AzureKeyCredential.
 - Support for creating SmsClient with TokenCredential
+- Added support for 1:N SMS messaging
+- Added support for SMS idempotency
```

Comment:
Missing `.` at the end. Do you mind fixing line 7 as well?

- Send method series in SmsClient are idempotent under retry policy.
- Added support for tagging SMS messages.